setup.py

```diff
@@ -6,7 +6,7 @@
 
 try:
```

Comment:
Since, in all the other places you haven't used `try - except` block to import `input`, you can directly use,
```python
from builtins import input
```
(and remove try - except block)

Reply:
Sure, I will do it